src/types/associatedArtifact.types.ts

```diff
@@ -1,7 +1,8 @@
 import { Uppy as UppyType } from '@uppy/core';
 import { CustomerRrsType } from './customerInstitution.types';
+import { FileType } from './registration.types';
 
-export type AssociatedFileType = 'PublishedFile' | 'UnpublishedFile' | 'UnpublishableFile';
+export type AssociatedFileType = FileType.PublishedFile | FileType.UnpublishedFile | FileType.UnpublishableFile;
```

Comment:
Er det ikke naturlig å erstatte filtypen her?
`FileType` er nå definert i `registration.types.ts`, uten at den brukes i den fila, så er denne typen i praksis bare en manuell kopi av enumen til å bli en type

```suggestion
export enum FileType {
  UnpublishedFile = 'UnpublishedFile',
  UnpublishableFile = 'UnpublishableFile',
  PublishedFile = 'PublishedFile',
}
```